src/test/java/racingcar/CarTest.java

```diff
@@ -0,0 +1,40 @@
+package racingcar;
+
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class CarTest {
+
+    @Test
+    @DisplayName("객체 생성 테스트")
+    void create() {
+        assertDoesNotThrow(() -> {
+            Car car = new Car();
+        });
```

Comment:
```suggestion
        assertDoesNotThrow(() -> new Car());
```
한줄로도 표현 가능하겠네요! 따로 변수에 할당할 필요는 없어보입니다. 참고만 부탁드려요!